common/adminBlock/EntryAddAuditServer.go

```diff
@@ -37,6 +39,7 @@ func (e *AddAuditServer) String() string {
 	return (string)(out.DeepCopyBytes())
 }
 
+// UpdateState updates the factomd state to include the new audit server information
 func (c *AddAuditServer) UpdateState(state interfaces.IState) error {
```

Comment:
(c *AddAuditServer) should be (e *AddAuditServer)

Reply:
Not sure how i missed this one. :-)